From 73fc179db7cd5fc88c5f2298d97dcc5bf3c88caf Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 15 May 2021 12:00:45 +0100
Subject: [PATCH] gnu: static-dispatch: Update to 0.5.1

* gnu/packages/lisp-xyz.scm (sbcl-static-dispatch):
  [inputs]: Replace trivia to optima
---
 gnu/packages/lisp-xyz.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 172c5d9805..9d07d5b058 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -13169,7 +13169,7 @@ environment access API.")
 (define-public sbcl-static-dispatch
   (package
     (name "sbcl-static-dispatch")
-    (version "0.5")
+    (version "0.5.1")
     (source
      (origin
        (method git-fetch)
@@ -13178,7 +13178,7 @@ environment access API.")
              (commit (string-append "v" version))))
        (file-name (git-file-name "static-dispatch" version))
        (sha256
-        (base32 "1a4vgfcn6qnpdkxxv93rri7zf3c92alixp6nyqwpz8mg2xmvy1j3"))))
+        (base32 "1602vx6ybp0n8mbrrp6q8397fkkyvhrqpahc302pjdb57qildajz"))))
     (build-system asdf-build-system/sbcl)
     (native-inputs
      `(("fiveam" ,sbcl-fiveam)))
@@ -13190,7 +13190,7 @@ environment access API.")
        ("cl-environments" ,sbcl-cl-environments)
        ("closer-mop" ,sbcl-closer-mop)
        ("iterate" ,sbcl-iterate)
-       ("trivia" ,sbcl-trivia)))
+       ("optima" ,sbcl-optima)))
     (home-page "https://github.com/alex-gutev/static-dispatch")
     (synopsis "Static generic function dispatch for Common Lisp")
     (description "Static dispatch is a Common Lisp library, inspired by
-- 
2.31.1

